events/models.py

```diff
@@ -5,9 +5,23 @@
 from django.forms import ModelForm
 from django.utils.translation import ugettext_lazy as _
 from django.utils.encoding import smart_str, smart_unicode
+from django.template import defaultfilters
 
 from django_countries.countries import COUNTRIES
 
+class Category(models.Model):
```

Comment:
@mapeveri could you please add a `Meta` nested class to `Category` and define its `verbose_name` and `verbose_name_plural` to avoid typos in the admin such as *Categorys* instead of *Categories*.